app/assets/stylesheets/barnardos/_textarea.scss

```diff
@@ -23,21 +23,28 @@
   color: $black;
   font-weight: normal;
   margin-top: $gutter / 2;
+  position: relative;
+
+  .has-error > & {
```

Comment:
Doesn't the `&` mean "parent selector"? I'm a little curious. What does this mean in this context? Not easily googleable 😄

Reply:
For SASS, the `&` is used when nested to define where the parent class is placed. This will output in the CSS:

```css
.has-error > .textarea__label {
  ...
}
```

You can read some tips about it here: https://css-tricks.com/the-sass-ampersand/
This is also a decent tool for testing out fun SASS things: https://www.sassmeister.com/

Reply:
Did *not* know how powerful this was, thanks. I've only been using it to refer to the parent in inner blocks, had no idea you could use it like [this](http://www.joeloliveira.com/2011/06/28/the-ampersand-a-killer-sass-feature/)

Reply:
It is brilliant for BEM structure as you can do:

```scss
.block {
  display: block;

  &__element {
    color: blue;

    &--modifier {
      background: orange;
    }
  }
}
```

Which will output:

```css
.block {
  display: block;
}

.block__element {
  color: blue;
}

.block__element--modifier {
  background: orange;
}
```

Reply:
Some devs discourage use of that notation with BEM because they would rather you don't nest (as you don't need to) and having the full version makes it easier to find but I'm all for what is practicle.